dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClasspathChecker.java

```diff
@@ -49,33 +49,43 @@
 
   private static final Logger logger = Logger.getLogger(ClasspathChecker.class.getName());
 
-  public static ClasspathChecker create(
-      boolean onlyReachable, List<Path> jarPaths, Iterable<Path> entryPoints)
+  public static ClasspathChecker create(List<Path> jarPaths, Iterable<Path> entryPoints)
       throws IOException {
     Preconditions.checkArgument(
         !jarPaths.isEmpty(),
         "The linkage classpath is empty. Specify input to supply one or more jar files");
     ClassDumper dumper = ClassDumper.create(jarPaths);
-    return new ClasspathChecker(onlyReachable, dumper, entryPoints);
-  }
 
-  /**
-   * If true, the report excludes linkage errors on classes that are not reachable
-   * from the entry points of the class usage graph.
-   */
-  private final boolean reportOnlyReachable;
+    ImmutableMap.Builder<Path, SymbolReferenceSet> jarToSymbolBuilder = ImmutableMap.builder();
```

Comment:
Check out `Maps.toMap(jarPaths, ClassDumper::scanSymbolReferencesInJar)`.

Reply:
Hmm. `IOException` prevents it. 

![image](https://user-images.githubusercontent.com/28604/52360052-1c564980-2a09-11e9-8225-fe14a380b448.png)